task2.cpp

```diff
@@ -0,0 +1,175 @@
+#include<iostream>
+#include<fstream>
+#pragma warning (disable: 4996)
+using namespace std;
+const char fileName[] = "offers.bin";
+const char a = 'a';
+const char i = 'i';
+const char s = 's';
+const char f = 'f';
+const char q = 'q';
+struct Offer {
+	char firmName[25];
+	int coworkers;
+	int offDays;
+	long long payment;
+};
+
+size_t getFileSize(ifstream& f) {
+	size_t currentPos = f.tellg();
+	f.seekg(0, ios::end);
+	size_t size = f.tellg();
+
+	f.seekg(currentPos);
+	return size+1;
```

Comment:
why +1?

Reply:
It's a mistake I intended add 1 to the functions where I read offers from the file because for some reason when I use getFilesSize(in)/sizeof(Offer) I get one less offer.